docs/language/interface_libraries.rst

```diff
@@ -37,9 +37,9 @@ When writing libraries there are restrictions compared to contracts:
 .. note::
 
     When using the Ethereum Foundation Solidity compiler, library are a special contract type and libraries are
-    called using `delegatecall`. Parity Substrate has no ``delegatecall`` functionality so Solang statically
```

Comment:
@seanyoung WDYT, now that we have that should we change it to align with with `solc`? I think that inlining them makes much more sense economically anyways. However, we should also try do be as close to EVM Solidity as possible; are there further implications in-lining them?

Reply:
Ah yes thanks for pointing this out. I need to give this a more close look in the near future. Ideally we can have the best of both worlds. I think if not inlined, the libraries could be instantiated in the storage initializers (or some similar mechanism).